courses/free-courses-pt_BR.md

```diff
@@ -137,6 +137,7 @@
 * [Curso completo e atual de HTML5 e CSS3 - Módulo 4 de 5](https://www.youtube.com/playlist?list=PLHz_AreHm4dkcVCk2Bn_fdVQ81Fkrh6WT) - Gustavo Guanabara (Curso em Vídeo)
 * [Introdução à linguagem CSS](https://www.udemy.com/introducao-a-linguagem-css/) - Diego Mariano (Udemy)
 * [Introdução à Linguagem HTML](https://www.udemy.com/introducao-a-linguagem-html/) - Diego Mariano (Udemy)
+* [Discover](https://app.rocketseat.com.br/discover) - Rocketseat
```

Comment:
Arrange the contents in alphabetical order.
[Discover]    **should come before**   [Introdução à linguagem CSS]